seminars/practice_2_string.py

```diff
@@ -53,13 +53,13 @@ def multiply_string(input_string: str, how_many: int) -> str:
     display the given string the number of times given in the `how_many`.
     """
     # student realisation goes here
-
+    print(input_string * how_many)
```

Comment:
Please also do not push modifications in the files other than in the `lab_1_classify_by_unigrams`. 
Again, this will not be merged.